packages/geo-tools/public/locales/en/GeoTools.json

```diff
@@ -1,6 +1,6 @@
 {
   "geoAddressSearch": {
-    "label": "Address Search",
+    "label": "Search Address",
```

Comment:
Let's hold this until I fix the current publishing issues after @aruniverse's fix.  I'll have a PR up in a couple minutes.

Reply:
Once #87 is merged this should be good to go.